docker-compose.yml

```diff
@@ -3,19 +3,19 @@ services:
   db:
     container_name: respa-db
     build: ./docker/postgres/
-    restart: always
+    restart: on-failure
     environment:
         POSTGRES_USER: respa
         POSTGRES_PASSWORD: respa
         POSTGRES_DB: respa
     volumes:
-      - db_data:/var/lib/postgresql
+      - db_data:/var/lib/postgresql/data
   api:
```

Comment:
This is probably OK, but I'm not sure if it affects anything.

Reply:
What is not persisted anymore?

Reply:
without this change every time you stop and start the container, the db data is gone. the container creates a new and empty db.